Engine Hacks/SkillSystem/Other Skills/shadowgift.s

```diff
@@ -2,23 +2,29 @@
 .equ LuminaID, ShadowgiftID+4
 .equ ShadowgiftStaffOption, LuminaID+4
 .equ LuminaStaffOption, ShadowgiftStaffOption+4
+.equ ItemTable,LuminaStaffOption+4
 .thumb
 
-push	{r4-r6}
+push	{r14}
```

Comment:
But later you're overwriting r4-r6. In this case it happens to work fine because you're writing the same values to the same registers but you know that's bad in general for multiple reasons.